browser/extensions/api/brave_rewards_api.h

```diff
@@ -506,6 +506,18 @@ class BraveRewardsUpdatePrefsFunction : public ExtensionFunction {
   ResponseAction Run() override;
 };
 
+class BraveRewardsRequestAdsEnabledPopupClosedFunction
```

Comment:
I think we should remove `PopupClosed` because it confuses things imo. That fact that it gets triggered when the popup is closed is an implementation detail of the extension code.